Makefile

```diff
@@ -36,6 +36,11 @@ fast_tests:
 	python -m pip install .[tests]
 	python -m pytest tests/test_gaudi_configuration.py tests/test_trainer_distributed.py tests/test_trainer.py tests/test_trainer_seq2seq.py
 
+# Run unit tests related to zero shot object detection
+fast_tests_zeroshotobjectdetection:
+	python -m pip install .[tests]
+	python -m pytest tests/test_zero_shot_object_detection.py
```

Comment:
Let's not add that now, I'll think of a command that enables to test all the examples that were added recently

Reply:
Yes, this PR covers pretty much everything:
- On the model side:
  - add the new model to the table in the README and in the doc
  - monkey patching in `modeling_utils.py`
- On the example side:
  - add a README with an example command
  - add a test